include/micm/solver/solver.hpp

```diff
@@ -80,6 +81,23 @@ namespace micm
       return solver_.Solve(time_step, state, params);
     }
 
+    /// @brief Returns the maximum number of grid cells per state
+    /// @return Number of grid cells
+    /// @details This is the maximum number of grid cells that can that fit
+    ///          within one group for vectorized solvers. For non-vectorized solvers,
+    ///          there is no limit other than the maximum size of a std::size_t.
```

Comment:
```suggestion
    /// @brief Returns the maximum number of grid cells per state
    /// @return Number of grid cells
    /// @details This is the maximum number of grid cells that can fit
    ///          within one group for vectorized solvers. For non-vectorized solvers,
    ///          there is no limit other than the maximum size of a std::size_t.
```
I am curious, in what occasion do you need to call this function?

Reply:
I noticed it would be useful in the MUSICA library to know how to create a state with the number of grid cells equal to the vector dimension of the vectorized matrices. There wasn't an easy way to access this information before.

Reply:
I see, thanks!